cmd/thanos/query.go

```diff
@@ -294,6 +294,13 @@ func runQuery(
 		api := v1.NewAPI(logger, reg, engine, queryableCreator, enableAutodownsampling)
 		api.Register(router.WithPrefix("/api/v1"), tracer, logger)
 
+		router.Get("/-/healthy", func(w http.ResponseWriter, r *http.Request) {
+			w.WriteHeader(http.StatusOK)
+			if _, err := fmt.Fprintf(w, "Thanos is Healthy.\n"); err != nil {
```

Comment:
```suggestion
			if _, err := fmt.Fprintf(w, "Thanos Querier is Healthy.\n"); err != nil {
```
.. to be specific (: